### DISCUSSION TITLE
Payment not received since almost half a year

### DISCUSSION THREAD

**Discussion by ishan-surana** [community/community#147333] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I had been sponsored on a one time basis as a test on June 30th, and while the sponsor activity feed shows the history of sponsorship, the Stripe has shown nothing related to it. Nor is the Payouts overview on GitHub showing anything related to it. Stripe isn't showing anything owed to me,  when I should have been eligible for a $1 payout. Please help me with this issue.

## Related Images
**Sponsorship history**
![image](https://github.com/user-attachments/assets/f4b797f5-4500-415a-a8f1-180a76bac714)
**Stripe**
![image](https://github.com/user-attachments/assets/e92cdb86-4765-4065-bb92-71bb88f8bfbe)
**GitHub Payouts overview**
![image](https://github.com/user-attachments/assets/ac13baff-9195-450a-abcc-cdccb025ba43)

**Comment by mecodeatlas** [comment]
Hi @ishan-surana! The best place to go for answers specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
